### PR TITLE
Cleanup code in WinForms Data Visualization

### PR DESCRIPTION
**Pull request by paul1956**
I did a basic code cleanup (no Logic changes), feel free to ignore/close if you are not interested.

### CODE REVIEW

**Comment by kirsan31** [comment]
Omg - lots of changes to review  😱 I will look into it, thank you @paul1956.

P.s. have you tried the latest V.S. preview (17.4 P5) in terms of #4? [There is a discussion ](https://github.com/KlausLoeffelmann/NetControlDesigners/pull/9#issuecomment-1293719751)on the topic of the designer with Klaus, I hope we will come to something in the end...

File: src/System.Windows.Forms.DataVisualization/Utilities/XmlSerializer.cs [hunk 1]
```diff
@@ -907,23 +907,23 @@ internal bool IsSerializableContent(string propertyName, object parent)
 			bool	serializable = true;
 			if(_serializableContent.Length > 0 || _nonSerializableContent.Length > 0)
 			{
-				int		serialzableClassFitType = 0;	// 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
-				int		serialzablePropertyFitType = 0;	// 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
+                // 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
+                // 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
```
Comment (kirsan31) [inline src/System.Windows.Forms.DataVisualization/Utilities/XmlSerializer.cs:911]: Lost the meaning of the comments.

File: src/System.Windows.Forms.DataVisualization/Utilities/XmlSerializer.cs [hunk 2]
```diff
@@ -907,23 +907,23 @@ internal bool IsSerializableContent(string propertyName, object parent)
...
+                serializable = IsPropertyInList(GetSerializableContentList(), ownerClassName, propertyName, out int serialzableClassFitType, out int serialzablePropertyFitType);
 
 				// Check if property in this class is part of the NON serializable content
 				if(serializable)
 				{
-					int		nonSerialzableClassFitType = 0;	// 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
-					int		nonSerialzablePropertyFitType = 0;	// 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
-					bool	nonSerializable = IsPropertyInList(GetNonSerializableContentList(), ownerClassName, propertyName, out nonSerialzableClassFitType, out nonSerialzablePropertyFitType);
+                    // 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
+                    // 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
```
Comment (kirsan31) [inline src/System.Windows.Forms.DataVisualization/Utilities/XmlSerializer.cs:921]: Lost the meaning of the comments.

**Review by kirsan31 — CHANGES_REQUESTED** [review]

File: src/System.Windows.Forms.DataVisualization/General/Legend.cs [hunk 3]
```diff
@@ -1738,7 +1714,7 @@ private int GetMaximumNumberOfRows(int[] rowsPerColumn)
                     Size cellSize = legendCell.MeasureCell(
                         graph,
                         fontSizeReducedBy,
-                        (this.autofitFont == null) ? this.Font : this.autofitFont,
+this.autofitFont ?? this.Font,
```
Comment (kirsan31) [inline src/System.Windows.Forms.DataVisualization/General/Legend.cs:1717]: Broken indentation.

File: src/System.Windows.Forms.DataVisualization/Utilities/XmlSerializer.cs (hunk 1 above)
Comment (kirsan31) [reply src/System.Windows.Forms.DataVisualization/Utilities/XmlSerializer.cs:900]: This comments was for:
`serialzableClassFitType`
and
`serialzablePropertyFitType`

File: src/System.Windows.Forms.DataVisualization/Utilities/XmlSerializer.cs (hunk 2 above)
Comment (kirsan31) [reply src/System.Windows.Forms.DataVisualization/Utilities/XmlSerializer.cs]: This comments was for:
`nonSerialzableClassFitType`
and
`nonSerialzablePropertyFitType`

**Review by kirsan31 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -368,3 +368,4 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+/src/System.Windows.Forms.DataVisualization/ChartWin.csproj
```
[the remark this answers is not in the document]
Comment (paul1956) [reply .gitignore:371]: I don't know how; sorry those changes were supported to le local to my machine.

[the remark this answers is not in the document]
Comment (paul1956) [reply .gitignore]: I figured it out I need to remove the project from my solution so it should happen again.

**Comment by kirsan31** [comment]
@paul1956 thank you, we have only this 2 remaining:
https://github.com/kirsan31/winforms-datavisualization/pull/6#discussion_r1008681442
https://github.com/kirsan31/winforms-datavisualization/pull/6#discussion_r1008881965

P.s. I could fix these little things myself if you allowed editing from maintainers when creating the PR.

File: src/System.Windows.Forms.DataVisualization/General/Legend.cs (hunk 3 above)
Comment (paul1956) [reply src/System.Windows.Forms.DataVisualization/General/Legend.cs:1717]: Fixed

File: src/System.Windows.Forms.DataVisualization/ChartTypes/ColumnChart.cs [hunk 5]
```diff
@@ -843,24 +837,28 @@ private void ProcessChartType3D( bool labels, bool selection, ChartGraphics grap
...
+        area.Area3DStyle.LightStyle,
+        point.Color,
+        topDarkening,
+        bottomDarkening,
+        point.BorderColor,
+        point.BorderWidth,
+        point.BorderDashStyle,
+        barDrawingStyle,
+        true,
+        drawingOperationType);
```
[the remark this answers is not in the document]
Comment (paul1956) [reply src/System.Windows.Forms.DataVisualization/ChartTypes/ColumnChart.cs]: fixed

**Review by kirsan31 — APPROVED** [review]

**Comment by kirsan31** [comment]
Thanks @paul1956 👍

**Comment by paul1956** [comment]
No idea how to do that. My Git experience in limited to what I find in Google and what VS let's me do. I will try to find out how to change today. I skipped doing anything with pattern matching, are you interested in those cleanups?

**Comment by kirsan31** [comment]
> No idea how to do that. My Git experience in limited to what I find in Google and what VS let's me do. I will try to find out how to change today. 

I do not quite understand what is at stake. I already merged this PR - everything is fine 👍

Oh, if you are talking about this:

> P.s. I could fix these little things myself if you allowed editing from maintainers when creating the PR.

Then it's just one tick when creating a PR: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

> I skipped doing anything with pattern matching, are you interested in those cleanups?

Thank you, but I think we need to take a break  😂 - it's very tedious to check such a massive code changes (which is why I didn't do it myself in the first place 🙄).

**Pull request merged**